gtfu/src/main/java/gtfu/RealtimeVisualization.java

```diff
@@ -752,12 +752,7 @@ public static void main(String[] arg) {
         }
 
         if (list == null) {
-            try {
-                InputStream is = RealtimeVisualization.class.getResourceAsStream("/conf/agency-data.json");
-                list = (AgencyData[])Util.readJSONObjectFromStream(is, AgencyData[].class);
-            } catch (Exception e) {
-                throw new Fail(e);
-            }
+            usage();
```

Comment:
I think your idea of using AgencyYML is very interesting. You would have to iterate over the returned data and filter for agencies that have both a static and an RT feed, but it would a) expand the scope of agencies we can visualize traffic for, and b) it would be more up-to-date than a static file. If you don't feel like doing that now, please create an issue for it.

Reply:
Makes sense -  it would require adding GRaaS RT URL to agencies.yml, which is something we'd need to talk to with the file owners.

Created https://github.com/cal-itp/graas/issues/25

Reply:
It's already in there. Excerpt for CAE:

```
agency_name: Clean Air Express
  feeds:
    - gtfs_schedule_url: http://www.cleanairexpress.com/GTFS/GTFS.zip
      gtfs_rt_vehicle_positions_url: null
```